deploy/evmos/210_permissionless_deployments.ts

```diff
@@ -0,0 +1,27 @@
+import dotenv from "dotenv"
+import { DeployFunction } from "hardhat-deploy/types"
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { deployPermissionlessPoolComponents } from "../deployUtils"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  dotenv.config()
+  const { deployments, getNamedAccounts } = hre
+  const { get } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  deployPermissionlessPoolComponents(
+    hre,
+    (await get("SaddleTBTCMetaPoolDeposit")).address,
```

Comment:
Could we manually copy and save this file as `MetaSwapDeposit.json` ? Same for other networks that have it deployed under a different name.

Then we can verify they are all the same copies by comparing the bytecode in each json. (if not, we may need to re-deploy some of them)